src/gui/windows/debug.zig

```diff
@@ -61,6 +62,16 @@ pub fn render() void {
 		y += 8;
 		draw.print("Queue size: {}", .{main.threadPool.queueSize()}, 0, y, 8, .left);
 		y += 8;
+		if (main.frameCount % main.perfUpdateFrequency == 0)
+			main.lastPerformance = main.threadPool.performance.readAndReset();
+		const perf = main.lastPerformance;
+		const values = comptime std.enums.values(TaskType);
+		inline for(values) |t| {
+			const name = @tagName(t);
+			const i = @intFromEnum(t);
+			draw.print("    " ++ name ++ " time: {} ms/frame ({} µs/task)", .{@divFloor(perf.utime[i], 1000*main.perfUpdateFrequency), @divFloor(perf.utime[i], @max(1, perf.tasks[i]))}, 0, y, 8, .left);
```

Comment:
Better use `draw.print("    {s} time: ...", .{name, ...})` to reduce the comptime code generation (format strings are passed at comptime)